spec/models/job_proxy_dispatcher_spec.rb

```diff
@@ -176,6 +176,7 @@
     context "with container and vms jobs" do
       before(:each) do
         @jobs = (@vms + @repo_vms).collect(&:raw_scan)
+        User.current_user = FactoryGirl.create(:user, :userid => 'bob')
```

Comment:
Is this user's userid being set to "bob" important for the success of these tests?